src/test/java/uk/gov/digital/ho/hocs/queue/common/MigrationQueueListenerTest.java

```diff
@@ -0,0 +1,49 @@
+package uk.gov.digital.ho.hocs.queue.common;
```

Comment:
This file wouldn't be required I wouldn't have thought.

Reply:
Removed